.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Title of the PR
```

Comment:
Why is the title of the PR needed in the description of the PR ? I mean, it's already above it 🙈